### ISSUE TITLE
Support for Umami Tags

### ISSUE THREAD

**Issue by melchiorb** [ijkml/nuxt-umami#123]
Hi,

I was wondering if there are plans to directly support Umami tags as described here: https://umami.is/docs/tags

I could not find any mention in the configuration but maybe I'm missing something.

Thank you

**Comment by ijkml** [comment]
Hey :wave:. No, it's currently not implemented. Should be, soon.

Thinking about it, not sure whether to implement it as a param on the tracking fns:

```ts
umTrackEvent(eventName, eventData, tag?)
```

or an env variable that gets automatically added to all payloads

```ts
{
  ...payload,
  tag: process.env.NUXT_UMAMI_TAG,
}
```

Have you used tags in other projects? If yes, how do you use it?
I don't have ~much~ any experience with a/b testing, I'd appreciate your insight.

**Comment by melchiorb** [comment]
Hi,
thanks for the quick response.

In raw Umami I set it on the umami script tag, which would match your second idea. I use it to differentiate whether users are logged in or not, since this is important to interpret results.

There seems to be some overlap with the session identify method, but that one is only shown in the session itself. Tags can be used in most screens for filtering and such which is really useful for any segmenting where you need to be able to compare.

Hope this helps

**Comment by ijkml** [comment]
Hi again, @melchiorb.

This is available now in `v3.1.1`.  Ran into some weird issues, sorry it took a whole week!

Fire me an issue if anything isn't working as intended.
Happy weekend!
